docs/SETUP.md

```diff
@@ -279,40 +202,6 @@ red circle right now. Once your peer adds a matching trustline (a trustline
 with your ILP Kit as the hostname and the same currency), the status will turn
 green. Now you can send payments between each other's ledgers.
 
-## Launch ILP Kit
-
-You can [create a systemd](#systemd-setup) config for ILP Kit, but I'm just going to use [`nohup`](https://en.wikipedia.org/wiki/Nohup) to
-run ILP Kit independent of our SSH session.
-
-Create a file called `start.sh`, and put the following lines into it:
```

Comment:
Looks like this part of the instructions was accidentally deleted in your branch